solr/core/src/java/org/apache/solr/core/ConfigSetService.java

```diff
@@ -346,7 +342,7 @@ protected IndexSchema createIndexSchema(
    * Returns a modification version for the schema file. Null may be returned if not known, and if
    * so it defeats schema caching.
    */
-  protected abstract Long getCurrentSchemaModificationVersion(
+  public abstract Long getCurrentSchemaModificationVersion(
```

Comment:
This was made public so that a custom ConfigSetService can delegate this method call.